root/etc/e-smith/events/actions/nethserver-ejabberd-backup

```diff
@@ -20,4 +20,11 @@
 # along with NethServer.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-/opt/ejabberd-*/bin/ejabberdctl backup /var/lib/nethserver/ejabberd/ejabberd.backup
+# Test if the service is running before to backup
+/usr/bin/systemctl is-active --quiet ejabberd
+if [[ $? -eq 0 ]]; then
```

Comment:
What about an "exit early" style? Is it more readable?

i.e.

```text
if guardcond1 then
  exit 0 // nothing to do
fi

if guardcond2 then
   exit 0  // nothing to do
fi

// ...do things

```

Reply:
Ok to change the style condition, just a matter of taste :)

I rather prefer to test if the service is running because you might have a status enabled but ejabberd down.

If ejabberd is down, the pre-backup will fail

Reply:
Ok! Go with it!